Assets/Prefabs/Ball.prefab

```diff
@@ -98,12 +98,13 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   paddle: {fileID: 0}
   hasStarted: 0
-  launchVelocity: {x: 2, y: 15}
-  bounceSounds:
-  - {fileID: 8300000, guid: 5cd521ee46ff3f54ea32bf02aef0fec0, type: 3}
-  - {fileID: 8300000, guid: c22a79a9053d9f849960377adb8b0f37, type: 3}
-  - {fileID: 8300000, guid: b5c32c79340bf5642b0e454004d80811, type: 3}
```

Comment:
Is there a reason for removing bounce sounds from the Ball prefab? I think they should remain there.

Reply:
I fixed the sound references, I think they got killed accidentally in the merge. Prefab changes aren't as easy to follow as changes to the code.